python/grass/script/core.py

```diff
@@ -2054,6 +2059,74 @@ def create_environment(gisdbase, location, mapset, env=None):
     return f.name, env
 
 
+def find_addon_name(addons, url=None):
+    """Find correct addon name if addon is a multi-addon
+    e.g. wx.metadata contains multiple modules g.gui.cswbrowser etc.
+
+    Examples:
+    - for the g.gui.cswbrowser module the wx.metadata addon name is
+    returned
+    - for the i.sentinel.download module the i.sentinel addon name is
+    returned
+    etc.
+
+    :param list addons: list of individual addon modules
+    :param str url: Addons modules.xml file URL
+
+    :return: list of unique simple and multi addons
+
+    >>> addons = find_addon_name(
+    ...    addons=[
+    ...             "g.gui.metadata",
+    ...             "g.gui.cswbrowser",
+    ...             "db.csw.run",
+    ...             "db.csw.harvest",
+    ...             "db.csw.admin",
+    ...             "v.info.iso",
+    ...             "r.info.iso",
+    ...             "t.info.iso",
+    ...             "db.join",
+    ...   ]
+    ... )
+    >>> addons.sort()
+    >>> addons
+    ['db.join', 'wx.metadata']
+    """
```

Comment:
Not sure I understand this example result, why only those 2 addons?

Reply:
> Not sure I understand this example result, why only those 2 addons?

wx.metadata is multi-addon (contains g.gui.metadata addon, g.gui.cswbrowser addon and etc.), we need identify multi-addon name which is in this example wx.metadata if user want uninstalling multi-addon (see GUI Manage installed extension dialog list of installed addon after you install some multi-addon e.g. wx.metadata)